src/ui/panels/quests.js

```diff
@@ -158,7 +158,7 @@ export class Quests {
             let sceneText = this.statsScrollWindow.add
                 .text(0, 0, statTexts[statkey], {
                     fill: "white",
-                    fontSize: 11,
+                    font: "14px runescape",
```

Comment:
I think bumping it up to 16 or 16.5 would be good. Also btw if it's not too wide I'm pretty sure you can put "white" into font rather than having it separate.

Reply:
In addition maybe separate text and number text so that you can left align all the numbers. Then the numbers can be truncated by # of digits to fit within the space using k, m, b, etc. For easy reading maybe color the numbers orange or blue like the right click menu or chat menu.